### ISSUE TITLE
An extra role in UI Prototype

### ISSUE THREAD

**Issue by lstkz** [topcoderinc/cab#69]
What if we can have an extra member/role (similar to an architect in Assembly) that will extract all assets and icons from PSD?
Currently, all competitors must do the same work and obtain the same images. Sometimes it can be time-consuming because you must merge layers, hide some things before exporting.

The client cost can be still the same. We can reduce prize pool by ~20%, and the extra member will get that money.

I think current UI prototype contests have definitely too big scope, especially if we must implement directly in React or Angular and keep some standards.

**Comment by ThomasKranitsas** [comment]
Totally agree! Exporting all icons is a time consuming task.
It would be great if we had another phase where a member will have to extract all assets (icons, patterns etc) from psd files so prototypers will not spend time to do that.

**Comment by lstkz** [comment]
We don't need an extra phase.
He can start extracting it 1 day before the contest starts and deliver missing or pending assets during the contest.

**Comment by dedywahyudi** [comment]
@lsentkiewicz 
This is good idea to let storyboard designer extract the assets, since designer should be familiar with the storyboard layer/structure. 

Might need educate them if Prototype require 1x, 2x, 3x image sizes. Also if need svg or other type files.

**Comment by nomo-kazza** [comment]
I agree with @dedywahyudi.
Even there are an option to extract easily in Photoshop CC if the layers and its name are arranged properly http://blogs.adobe.com/jkost/2014/10/new-extract-assets-for-generator-in-photoshop-cc.html

**Comment by birdofpreyru** [comment]
Good point! The input into UI-prototype content should be html specs exported from Sketch, along with all necessary assets, so that competitors can just focus on html/css/js, without a need to distract themselves for Photoshop manipulations.

**Comment by hokienick** [comment]
@adroc-tc to add to pricing planning for s/m/l prototypes.  In progress, will update next meeting.
